### PR TITLE
Explain Keys

### PR DESCRIPTION
**Pull request by vrotmanh**

### CODE REVIEW

File: KEYS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+There are 2 different Ethereum Keys, the first one is your **Ethereum Private Key**: this has to be private, nobody should have access to this but you. With this key you can sign transactions and add them to the blockchain. This is you authentication key, to prove your identity.
+
+The second one is your **Ethereum Address**: this is public and is how external users refer to yourself. If someone want's to share data with you using Linnia they will need your Ethereum Address.
+
+
+
+### Encryption Keys
+
+In order to add data to the Linnia Protocol, the data has to be encrypted. That means that the data will look like a bunch of random letters and numbers. When you encrypt some data you are hiding the real content so nobody can understand. In order to recover the original data you need to decrypt. There are also 2 **Encryption Keys**
```
Comment (MaxBlaushild) [inline KEYS.md]: Remove the sentence about random letters and numbers maybe?? I think the next sentence covers it

File: KEYS.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+This first one is the **Encryption Private Key**: similar to the **Ethereum Private Key** this key has to be private too and the owner is the only one that should have access to. You need this key in order to decrypt the data, so after you retrieve the hidden random looking data, with this key you will be able to convert that into the readable original content.
```
Comment (MaxBlaushild) [inline KEYS.md]: These two paragraphs are good, but replace the `:`s with `.`s

File: README.md [hunk 3]
```diff
@@ -34,3 +34,9 @@ Getting started building blockchain technology is tough! As with any emerging te
 ## Contributing
 
 Linnia prides itself on being an open source protocol. We greatly appreciate any contributions you can make to further the cause. Please reference our [contribution documentation](./CONTRIBUTING.md) before getting started!
+
+
+
+## What Keys Linnia Uses?
+
+Go [HERE](KEYS.md) to read more about the keys that you need to use Linnia Protocol
```
Comment (MaxBlaushild) [inline README.md]: `read more about encryption patterns and how the Linnia Protocol uses them.`

File: KEYS.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Keys
+
+The Linnia protocol need 4 different keys to work: (Ethereum Private Key, Ethereum Address, Encryption Private Key, Encryption Public key)
```
[the remark this answers is not in the document]
Comment (vrotmanh) [reply KEYS.md]: done

File: KEYS.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+
+
+### Ethereum Keys
+
+In order to add data to the Linnia Smart Contracts, the user needs an ethereum wallet to be able to sign transaction and add them to the ethereum blockchain. (We recommend to download Metamask https://metamask.io/ to create the **Ethereum Keys** and sign the transaction)
```
[the remark this answers is not in the document]
Comment (vrotmanh) [reply KEYS.md]: done

**Pull request merged**